app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,42 @@
+class CommentsController < ApplicationController
+  before_action :get_post, only: [:new, :create, :destroy]
```

Comment:
I would call this load_post to reflect what you're doing. "get" is a bit of an overloaded term.